apps/src/StudioApp.js

```diff
@@ -981,6 +1000,31 @@ StudioApp.prototype.onReportComplete = function (response) {
 };
 
 /**
+ * Given a level definition, do we want to show instructions in markdown form.
+ * @param {object} level
+ * @returns {boolean}
+ */
+StudioApp.prototype.isMarkdownMode = function (level) {
+  return window.marked && level.markdownInstructions &&
+    this.LOCALE === ENGLISH_LOCALE;
+};
+
+/**
+ * @param {object} level
+ * @returns {string} Level's markdown instructions, with the markdown processed
+ *   into a string if in markdownMode
+ */
+StudioApp.prototype.getMarkdownInstructions = function (level) {
+  if (!this.isMarkdownMode(level)) {
+    return level.markdownInstructions;
+  }
+
+  return marked(this.substituteInstructionImages(
+    level.markdownInstructions, {}));
```

Comment:
I'm fine with not-optional in this case, especially since (I believe) omitting the second argument or passing an empty object makes the method a no-op entirely?